hnclient-v2/src/main/java/ca/bc/gov/hlth/error/ErrorBuilder.java

```diff
@@ -0,0 +1,34 @@
+package ca.bc.gov.hlth.error;
+
+import java.sql.Timestamp;
+
+public class ErrorBuilder {
+
+	private static String HL7Error_Default = "MSH|^~\\\\&|UNKNOWNAPP|UNKNOWNCLIENT|HL7ERRORGEN|UNKNOWNFACILITY|||ACK|||2.4\n";
```

Comment:
I think we may to replace the UNKNOWN sections with values from the message that was sent in from the point of service.